src/shared/GlobalTopBanners/ProPlanFeedbackBanner/ProPlanFeedbackBanner.test.tsx

```diff
@@ -7,30 +7,22 @@ import React from 'react'
 import { MemoryRouter, Route } from 'react-router-dom'
 
 import { TrialStatuses } from 'services/account'
-import { TierNames } from 'services/tier'
 import { BillingRate, Plans } from 'shared/utils/billing'
 
 import ProPlanFeedbackBanner from './ProPlanFeedbackBanner'
 
-const mockProTier = {
-  owner: {
-    plan: {
-      tierName: TierNames.PRO,
-    },
-  },
-}
-
 const mockTrialData = {
   hasPrivateRepos: true,
   plan: {
     isEnterprisePlan: false,
     isFreePlan: false,
+    isProPlan: true,
     baseUnitPrice: 10,
     benefits: [],
     billingRate: BillingRate.MONTHLY,
     marketingName: 'Users Basic',
     monthlyUploadLimit: 250,
-    value: Plans.USERS_BASIC,
+    value: Plans.USERS_PR_INAPPM,
```

Comment:
mmm why did this value change from basic to pro?

Reply:
I think the original mock for this test was copy pasted from somewhere else, this is what the mock should've been to begin with

I only updated what I needed to tho admittedly